tests/functional/sync_v2_force_auth.js

```diff
@@ -21,7 +21,7 @@ define([
   var PASSWORD = '12345678';
   var url;
 
-  var listenForFxaCommands = FunctionalHelpers.listenForWebChannelMessage;
+  var noSuchBrowserNotification = FunctionalHelpers.noSuchBrowserNotification;
```

Comment:
Noticed that `noSuchBrowserNotification` takes a callback parameter, but never uses it. Could be removed.

https://github.com/mozilla/fxa-content-server/blob/issue-3431-force-auth-signs-out-of-sync/tests/functional/lib/helpers.js#L690